CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.20.1 (2021-07-16)
+
+- Fixed the selection box component to properly disable after init. [#280](https://github.com/blackbaud/skyux-forms/pull/280)
```

Comment:
Might just be me, but as written, this sounds like we've updated the component to always disable selection boxes after they initialize. I think a slightly longer changelog entry might be clearer here.
 ```suggestion
- Fixed the selection box component to properly style selection boxes that are disabled after initialization. [#280](https://github.com/blackbaud/skyux-forms/pull/280)
```